aten/src/ATen/native/quantized/cudnn/BinaryOps.cpp

```diff
@@ -242,7 +242,7 @@ Tensor add(Tensor qa, Tensor qb, double output_scale, int64_t output_zero_point)
       run(plan_desc);
       execution_plan_cache[key] = plan_desc;
       return quantized_output.view(orig_sizes);
-    } catch (cudnn_frontend::cudnnException &e) {std::cout << "cudnn error:" << e.what() << std::endl;} catch(c10::CuDNNError &e) { std::cout << "other error" << e.what() << std::endl;}
```

Comment:
What is the disadvantage of not printing out errors, would it still show cudnnException somewhere?

Reply:
These are somewhat expected and recoverable failures, if a plan fails we just go to the next one. At some point cuDNN claimed that these would not be common, but as https://[github.com/pytorch/pytorch/issues/126605](https://github.com/pytorch/pytorch/issues/126605) showed it is still a common issue that surprises users. These failures should not be visible if they are caught here.

Reply:
I see. Eventually these exceptions are just false alarms. Thanks for the explanation!